src/js/services/auth-service.js

```diff
@@ -0,0 +1,11 @@
+import { store } from 'stores/app-store';
+import { core } from 'services/core';
+
+import { setUser } from 'actions/user-actions';
+
+export function login(props) {
+    return core().appUsers.init(props).catch((e) => {
+        console.log(e); //eslint-disable-line no-console
```

Comment:
I added lint exceptions to these console.logs but should I have? Do these errors belong in the console?